### PR TITLE
Router Docs - Remove idea of sharing router outside of Redwood

### PR DESCRIPTION
**Pull request by andrew-hwahin**
Related to issue #1888

### CODE REVIEW

**Comment by andrew-hwahin** [comment]
@thedavidprice @dthyresson is this correct?

File: packages/router/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Redwood Router (RR from now on) is designed to list all routes in a single file,
 
 ## Installation
```
Comment (Tobbe) [inline packages/router/README.md:9]: I think this entire section can be removed now. I looked at our other README files, and they don't have installation sections.

What do you think?

Comment (mojombo) [reply packages/router/README.md:9]: Agreed, if you can only use it in a Redwood app, it's installed when you create the Redwood app. No need for this section at all.

Comment (dthyresson) [reply packages/router/README.md]: LGTM

**Review by thedavidprice — APPROVED** [review]
🚀

**Pull request merged**